cmd/pkg-unpack/main.go

```diff
@@ -17,63 +17,71 @@ limitations under the License.
 package main
 
 import (
+	"bufio"
+	"bytes"
 	"context"
 	"fmt"
+	"io"
 	"os"
 
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructuredscheme"
-
 	"github.com/pkg/errors"
 	"github.com/spf13/afero"
-	"gopkg.in/yaml.v2"
-	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructuredscheme"
+	"k8s.io/apimachinery/pkg/runtime/serializer/json"
+	"k8s.io/apimachinery/pkg/util/yaml"
 
 	"github.com/crossplane/crossplane-runtime/pkg/parser"
 )
 
 func main() {
-	if err := Run(context.Background()); err != nil {
+	be := parser.NewFsBackend(afero.NewReadOnlyFs(afero.NewOsFs()), parser.FsDir("."), parser.FsFilters(parser.SkipNotYAML()))
+	src, err := be.Init(context.Background())
+	if err != nil {
+		// NOTE(negz): fmt.Print prints to stderr.
```

Comment:
I believe `fmt.Print` will print to `stdout` but the builtin `print` will print to `stderr`.

https://golang.org/pkg/fmt/#Print
https://golang.org/pkg/builtin/#print

Reply:
Huh, weird. I doublechecked this when working on this PR. Must have been mistaken.